eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/admin/handler/AdminHandlerManager.java

```diff
@@ -117,7 +117,7 @@ public void registerHttpHandler() {
             eventMeshGrpcServer.getEventMeshGrpcConfiguration()));
     }
 
-    private void initHandler(HttpHandler httpHandler) {
+    public void initHandler(HttpHandler httpHandler) {
         EventMeshHttpHandler eventMeshHttpHandler = httpHandler.getClass().getAnnotation(EventMeshHttpHandler.class);
```

Comment:
Is it a good practice to change a private method's scope just because a test class needs to call it? Maybe we can try to use Reflection.